luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,113 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation
+from lib.model.seqr_mt_schema import BaseSeqrSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+
+class SeqrMitoVariantSchema(BaseSeqrSchema):
+
+    def __init__(self, *args, high_constraint_region, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._high_constraint_region = high_constraint_region
+
+    # Mitochondrial only fields
+    @row_annotation()
+    def gnomad_mito(self):
+        return self._selected_ref_data.gnomad_mito
+
+    @row_annotation()
+    def mitomap(self):
+        return self._selected_ref_data.mitomap
+
+    @row_annotation(name='APOGEE')
+    def mitimpact(self):
+        return self._selected_ref_data.mitimpact
+
+    @row_annotation(name='HmtVar')
+    def hmtvar(self):
+        return self._selected_ref_data.hmtvar
+
+    @row_annotation()
+    def helix(self):
+        return self._selected_ref_data.helix_mito
+
+    @row_annotation()
+    def common_low_heteroplasmy(self):
+        return self.mt.common_low_heteroplasmy
+
+    @row_annotation()
+    def hap_defining_variant(self):
+        return self.mt.hap_defining_variant
+
+    @row_annotation()
+    def mitotip_trna_prediction(self):
+        return self.mt.mitotip_trna_prediction
+
+    @row_annotation()
+    def high_constraint_region(self):
+        return hl.is_defined(self._high_constraint_region[self.mt.locus])
+
+    # Fields with the same names but annotated differently
+    @row_annotation()
+    def rsid(self):
+        return self.mt.rsid.find(lambda x: hl.is_defined(x))
```

Comment:
The `rsid` field has to be annotated differently because the `rsid` field is an array of strings in the mitochondrial MT while we just need a string.

Reply:
what are some examples of the rsids in the VCF? Do any of the rows have multiple rsids, or is it always a list of 1 element?

Reply:
Here are some examples of the rsid field:
![image](https://user-images.githubusercontent.com/33550059/157700792-c97ab9d2-3975-4def-8018-78ddd35a4e65.png)